bitnami/postgresql/templates/backup/pvc.yaml

```diff
@@ -3,7 +3,7 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if and .Values.backup.enabled (not .Values.backup.cronjob.storage.existingClaim) -}}
+{{- if and .Values.backup.enabled (not .Values.backup.cronjob.storage.existingClaim) (not .Values.backup.cronjob.storage.existingVolume) -}}
```

Comment:
Hi @bastiaanb,

Thank you for your contribution!
 
In this cases, we use `extraVolumes` and `extraVolumeMounts`. We may want to refactor this in order to keep Bitnami's conventions. We could define `.cronjob.extraVolumes` in case we want specific mounts for the cronjob. Also, the PVC activation should be independent on any extraVolumes (so we could add different mounts independently of the PVC mount point). May you want to update the implementation?

Reply:
Hi @aoterolorenzo, thanks for the feedback. 
It makes sense to align with Bitnami's conventions. I'll update the implementation.

Reply:
Hi @aoterolorenzo, I changed the implementation. 
I have added `.cronjob.extraVolumes` and `.cronjob.extraVolumeMounts`.
Had to add `.cronjob.storage.enabled` too, to optionally disable PVC creation.